### DISCUSSION TITLE
Intercept on input change

### DISCUSSION THREAD

**Discussion by ariefsn** [cypress-io/cypress#27913] [Questions and Help]
Hi,
is there a way to intercept req but not from element click but from input value change?
so when input value changed, it will has req to the API, and I need to intercept that.
```
      cy.intercept('**/domain-check/*').as('getDomainCheck')
      cy.get('input-domain')
        .clear()
        .type('some-domain')
      cy.wait('@getDomainCheck').then(() => { })
```
thanks.

**Comment by kchuist** [comment]
When you input an value, if it sends that POST request. (Check Network if the call is being made). Then You can intercept that POST call once the value is entered, and the POST call is sent.

**Reply by ariefsn** [reply]
@kchuist 
yea that what I think.
but its doesn't works.
is my code wrong?
it cannot get `@getDomainCheck` when `cy.wait`
